.github/workflows/slash-command-dispatch.yml

```diff
@@ -10,7 +10,8 @@ on:
     # Add "edited" type for test purposes. Where possible, avoid using to prevent processing unnecessary events.
     # types: [created, edited]
 
-permissions: {}
+permissions:
+  issues: write
```

Comment:
> The "Slash Command Dispatch" workflow needs the issue write permission to add reactions.
> 
> The "format command" workflow needs the content write permission to push changes to PRs.
> 
> Not tested, but they should work based on my understanding.

Not sure either, but see https://github.com/peter-evans/slash-command-dispatch/issues/376#issuecomment-2628908062? If this doesn't work, try `contents: write`.